docker/run_config.py

```diff
@@ -0,0 +1,22 @@
+import argparse
+import flags.flags_parser as flags_parser
+import importlib
+
+
+def main():
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('model_file', help='The model file to execute.')
+    parser.add_argument('flags_file', help='The flags_file file that is to be executed.')
+    parser.add_argument('remainder', nargs=argparse.REMAINDER)
```

Comment:
Please write a comment. It looks like it's a model hyperparameter.

Reply:
Done